distribution/docs/src/main/resources/_contents/_sources/registry-store-contents.adoc

```diff
@@ -0,0 +1,26 @@
+==== Registry Store
+
+[NOTE]
+====
+The Registry Store is currently marked *Experimental*. While functional and tested, it is subject to change or removal during the incubation period.
+====
+
+{description}
```

Comment:
Description does not render correctly
![image](https://cloud.githubusercontent.com/assets/8041246/18291046/d99964fa-743a-11e6-83a6-12cf2d3de521.png)